### PR TITLE
preparation for v7.0.0 release

### PR DESCRIPTION
**Pull request by grossmannmartin**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Release of `v7.0.0`
|New feature| No 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| Yes 
|Fixes issues| ... 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,52 @@ The changelog is generated during the release process using [ChangelogLinker](ht
...
+
+#### Changed
+
+- [#824] Elasticsearch contains products after "build-demo" target now, thanks to [@pejaycz]
+
+### [shopsys/shopsys]
+
+#### Changed
+
+- [#845] shopsys dependencies now using caret carsion
```
Comment (vitek-rostislav) [inline CHANGELOG.md]: "carsion"? seems like some typo, could you please fix it in the PR title as well?

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,52 @@ The changelog is generated during the release process using [ChangelogLinker](ht
...
+- [#830] improved documentation "How to set up domains and locales"
+- [#782] changed forms in popup so they work with js validation
```
Comment (vitek-rostislav) [inline CHANGELOG.md]: I am really not sure where do some of the entries belong...For example, this particular one seems more like a fix in the framework or a change in the project-base :thinking: However, you do not need to change it now, let us discuss this on some of our meetings in general...

File: CHANGELOG.md [hunk 3]
```diff
@@ -27,6 +27,52 @@ The changelog is generated during the release process using [ChangelogLinker](ht
...
 <!-- changelog-linker -->
 
+## [v7.0.0](https://github.com/shopsys/shopsys/compare/v7.0.0-beta6...v7.0.0) - 2019-02-26
+
+### [shopsys/framework]
+
+#### Changed
+
+- [#817] now it is possible to run multiple cron commands
+- [#871] autoload helper classes from Tests\FrameworkBundle\Test namespace
```
Comment (vitek-rostislav) [inline CHANGELOG.md]: seems quite unimportant to me

File: docs/installation/installation-using-docker-linux.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Take a look at the article about [Monorepo](../introduction/monorepo.md) for mor
 ## Steps
 ### 1. Create new project from Shopsys Framework sources
 ```
-composer create-project shopsys/project-base --stability=beta --no-install --keep-vcs
+composer create-project shopsys/project-base --no-install --keep-vcs
```
Comment (vitek-rostislav) [inline docs/installation/installation-using-docker-linux.md]: please find "--stability=beta" in the all the md files in the project - there are still places where it was not removed (eg.there is a note with an explanation of the option below the command itself in each  of the guides, etc.)

File: CHANGELOG.md (hunk 1 above)
Comment (grossmannmartin) [reply CHANGELOG.md]: changed to "notation" 👍

File: CHANGELOG.md (hunk 3 above)
Comment (grossmannmartin) [reply CHANGELOG.md]: right, removed

**Pull request merged**